Solutions/CyberArkEPM/Parsers/CyberArkEPM.txt

```diff
@@ -0,0 +1,92 @@
+CyberArkEPM_CL
+| extend EventVendor = 'CyberArk',
+         EventProduct = 'Endpoint Privilege Manager',
+         EventSchemaVersion = '0.1',
+         EventCount=case(event_type_s == 'raw_event', agentEventCount_d, totalEvents_d),
+         EventMessage=case(event_type_s == 'raw_event', displayName_s, lastEventDisplayName_s),
+         ActingProcessFileInternalName=case(event_type_s == 'raw_event', fileName_s, lastEventFileName_s),
+         Justification=case(event_type_s == 'raw_event', justification_s, lastEventJustification_s),
+         EventSourceName=case(event_type_s == 'raw_event', sourceName_s, lastEventSourceName_s),
+         EventSourceType=case(event_type_s == 'raw_event', sourceType_s, lastEventSourceType_s),
+         ActorUsername=case(event_type_s == 'raw_event', userName_s, pack_array(firstEventUserName_s, lastEventUserName_s))
+| project-rename AccessAction=accessAction_s,
+                 AccessTargetName=accessTargetName_s,
+                 AccessTargetType=accessTargetType_s,
+                 AffectedComputers=affectedComputers_d,
+                 AffectedUsers=affectedUsers_d,
+                 AdminTaskId=adminTaskId_s,
+                 BundleId=bundleId_s,
+                 BundleName=bundleName_s,
+                 BundleVersion=bundleVersion_s,
+                 DvcId=agentId_g,
+                 AggregatedBy=aggregatedBy_s,
+                 AppType=applicationType_s,
+                 ApplicationSubType=applicationSubType_s,
+                 AppPackageDisplayName=appPackageDisplayName_s,
+                 CLSID=CLSID_s,
+                 ActingProcessFileCompany=company_s,
+                 DeceptionType=deceptionType_d,
+                 DefenceActionId=defenceActionId_d,
+                 EventType=event_type_s,
+                 EventSubType=eventType_s,
+                 Evidences=evidences_s,
+                 FileAccessPermission=fileAccessPermission_s,
+                 ActingProcessFileDescription=fileDescription_s,
+                 FileLocation=fileLocation_s,
+                 ActingProcessName=filePath_s,
+                 FileQualifier=fileQualifier_s,
+                 ActingProcessFileSize=fileSize_d,
+                 ActingProcessFileVersion=fileVersion_s,
+                 EventStartTime=firstEventDate_t,
+                 Hash=hash_s,
+                 JustificationEmail=justificationEmail_s,
+                 LastAgentId=lastAgentId_g,
+                 EventEndTime=lastEventDate_t,
+                 LogonAttemptTypeId=logonAttemptTypeId_d,
+                 LogonStatusId=logonStatusId_d,
+                 SrcFileMimeType=mimeType_s,
+                 ModificationTime=modificationTime_t,
+                 ActingProcessFileOriginalName=originalFileName_s,
+                 Owner=owner_s,
+                 PackageName=packageName_s,
+                 PolicyId=policyId_d,
+                 PolicyName=policyName_s,
+                 ActingProcessGuid=processCommandLine_g,
```

Comment:
Hey @averbn, in sample data there is not any guid used for "processCommandLine", will you please check and remove this, thanks